src/json-to-csv.ts

```diff
@@ -0,0 +1,106 @@
+import { format as dateFormat } from 'https://deno.land/std@0.184.0/datetime/mod.ts'
+import { stringify as csvStringify } from 'https://deno.land/std@0.184.0/csv/stringify.ts'
+import { Body, CinemaInfo, Movie, MovieHall } from './types.ts'
+import { metaDir } from './csv-to-ics.ts'
+
+const cfaApi = 'https://yt5.cfa.org.cn/api/libraryapi/arrangementPage'
+export const csvHeader = [
+  'name',
+  'englishName',
+  'year',
+  'cinima',
+  'playTime',
+  'endTime',
+]
+
+export function getFirstDayOfNextMonth(date: Date = new Date()): Date {
+  const readableMonth = date.getMonth() + 1
+  let readableMonthInNextMonth = readableMonth + 1
+
+  const lastDayInNextMonth = new Date(
+    date.getFullYear(),
+    readableMonth + 1,
+    0,
+  )
+
+  const yearInNextMonth = lastDayInNextMonth.getFullYear()
+
+  readableMonthInNextMonth = readableMonthInNextMonth > 12
```

Comment:
It can be solved with a single return line 😭  :
```
export function getFirstDayOfNextMonth(date: Date = new Date()): Date {
  return new Date(date.getFullYear(), date.getMonth() + 1, 1)
}
```